### PR TITLE
[plugin.video.noco] broken

### PR DESCRIPTION
**Pull request by gormux**
### Description
Service no longer exists.

### CODE REVIEW

File: plugin.video.noco/addon.xml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.noco" name="noco" version="1.0.16" provider-name="Julien Gormotte (gormux)">
+<addon id="plugin.video.noco" name="noco" version="1.0.17" provider-name="Julien Gormotte (gormux)">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
     <import addon="script.module.xbmcswift2" version="2.4.0"/>
     <import addon="script.module.requests" version="2.3.0"/>
   </requires>
+  <broken>service no longer exists</broken>
```
Comment (Rechi) [inline plugin.video.noco/addon.xml]: broken tag has to go into `<extension point="xbmc.addon.metadata">`

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

So please at least name your commit '[plugin.video.noco] broken'.

**Comment by gormux** [comment]
Sorry, this is the first time I use github to commit.
Is it ok now ?

**Comment by Rechi** [comment]
No as your PR now even has two commit and none has the correct commit message.

**Comment by enen92** [comment]
Cross-reference, closes https://github.com/xbmc/repo-plugins/issues/1997

**Pull request merged**